webapp/src/components/AcceptBidAssetPage/AcceptBidAssetPage.container.js

```diff
@@ -0,0 +1,72 @@
+import { withRouter } from 'react-router'
+import { connect } from 'react-redux'
+import { push } from 'react-router-redux'
+import { getAddress, isConnected } from '@dapps/modules/wallet/selectors'
+
+import { locations } from 'locations'
+import { ASSET_TYPES } from 'shared/asset'
+import { buildCoordinate } from 'shared/coordinates'
+import {
+  getMatchParams,
+  getMatchParamsCoordinates
+} from 'modules/location/selectors'
+import { acceptBidRequest, fetchBidByIdRequest } from 'modules/bid/actions'
+import {
+  getBidById,
+  isCancelIdle,
+  isLoading as isBidLoading
+} from 'modules/bid/selectors'
+import AcceptBidAssetPage from './AcceptBidAssetPage'
+
+const mapState = (state, ownProps) => {
+  let assetId
+  const { bidId, id } = getMatchParams(ownProps)
+  assetId = id
+
+  switch (ownProps.assetType) {
+    case ASSET_TYPES.parcel: {
+      const { x, y } = getMatchParamsCoordinates(ownProps)
+      assetId = buildCoordinate(x, y)
+      break
+    }
```

Comment:
I think we should be consistent between map dispatch and props, and I like the approach below.

It think it's clearer to have a case for estates here like:

```js
const matchParams = getMatchParams(ownProps)

// (...)
case ASSET_TYPES.estate: {
  assetId = matchParams.id
  break
}
```

instead of using id as default. You can use `matchParams.bidId` below